VAULT-34809 Utilization Report docs (#30203)
hashicorp/vault: website/content/api-docs/system/utilization-report.mdx

---
layout: api
page_title: /sys/utilization-report - HTTP API
description: The `/sys/utilization-report` produces a snapshot report of this cluster's utilization from all namespaces.
---

# `/sys/utilization-report`

@include 'alerts/enterprise-only.mdx'

@include 'alerts/restricted-admin.mdx'

> The `/sys/utilization-report` endpoint is a read-only endpoint that produces a snapshot
> report if this cluster's utilization from all namespaces, giving several high-level insights,
> including information about mounts, leases, lease count quotas, PKI, and KV secrets.

Reviewer: ```suggestion
The `/sys/utilization-report` endpoint returns a snapshot utilization report
for the targeted cluster. The report covers all namespaces and includes
high-level insights on mounts, leases, lease count quotas, PKI, and KV secrets.
```
Style correction: avoid possessives, 1 idea per sentence

## Get utilization report

> This endpoint produces a snapshot report of this cluster's utilization from all namespaces.

Reviewer: ```suggestion
Return a snapshot utilization report for all namespaces in the cluster.
```

Style correction: avoid "this" as a pronoun, avoid possessives
